Preference Access SSN Matching/LETTERSPECS/PREFACCESS.SSNMATCHING.CFG

```diff
@@ -0,0 +1,155 @@
+*  Line must begin with "ITLT:" (for loans)
+*  Line must begin with "ITET:" (for external loans)
+*
+*  Enter the Account, Share, Loan and or External Loan types which will exclude
+*  an Account, Share, Loan and/or External Loan from accessibility by this
+*  program. If left blank, Account, Share, Loan types and/or External Loan
+*  types will not be a consideration when analyzing target accounts.
+*
+*  VALID VALUES: range of values from 0-9999 (Account, Share, Loan)
+*                range of values from 1-8 (External Loan)
+*  DEFAULT: (left blank - Account, Share, Loan and/or External Loan
+*            types not a factor)
+ITAT:
+ITST:
+ITLT:
+ITET:
+*
+*------------------------------------------------------------------------------
+* EXCLUDE ACCOUNTS WITH INVALID SSNs
+*------------------------------------------------------------------------------
+*  Line must begin with "ISSN:"
+*
+*  Enter a list of invalid primary name record SSNs. List as a single
+*  comma-delimited list. Accounts where the primary member's SSN are
+*  found in this list will be skipped over. Enter up to 10 different
+*  SSNs. Use only numeric values.
+*
+*  VALID VALUES: Up to 10, comma delimited, 9-digit numeric values
+*  DEFAULT: [left blank - no invalid SSNs]
+ISSN:000000000,111111111,222222222,333333333,444444444,555555555,666666666,777777777,888888888,999999999
+*------------------------------------------------------------------------------
+* EXCLUDE SHARES WITH A NON-ZERO IRS CODE
+*------------------------------------------------------------------------------
+*  Line must begin with "ENZS:"
+*
+*  If TRUE, or left blank, target Shares with the SHARE:IRSCODE value>0
+*  will be excluded from analysis.
+*
+*  VALID VALUES: TRUE or FALSE
+*  DEFAULT: TRUE
+ENZS:TRUE
+*
+*------------------------------------------------------------------------------
+* VALID TARGET NAME TYPES
+*------------------------------------------------------------------------------
+* Line must begin with "VTNT:"
+*
+* Enter the valid name types the program should check on the targeted
+* accounts.
+*
+*  VALID VALUES: Range of values from 0-99 or 'ALL' (at creation,
+*                the max name type is 53, but this will allow for
+*                new name types in the future.
+*  DEFAULT: (left blank or 'ALL' - all name types will be checked)
+VTNT:ALL
+*
+*------------------------------------------------------------------------------
+* PREFERENCE ACCESS FLAGS TO SET
+*------------------------------------------------------------------------------
+* When a Preference Access record is created, which of these flags
+* should be set? At least one must be set to TRUE. If left blank, FALSE
+* will be assumed.
+*
+* Line must begin with "PAWF:" for the withdrawal flag.
+* Line must begin with "PADF:" for the deposit flag.
+* Line must begin with "PAIF:" for the inquiry flag.
+*
+* VALID VALUES: TRUE or FALSE
+* DEFAULT: TRUE
+PAWF:TRUE
+PADF:TRUE
+PAIF:TRUE
```

Comment:
Are there Situations in which for a Preference Access with Access Type 2 records where Enable Inquiries should be false?
I was under the impression that at least the Enable Inquiries needs to be true for parts of Online banking applications to work.

Reply:
I left it up for the CU to decide how they wanted the access flags set. They'll need to know how to have the flags set for their desired result.